src/Files.Shared/Files.Shared.csproj

```diff
@@ -12,6 +12,7 @@
     <ItemGroup>
         <!-- Do not update LiteDB to version 5 -->
         <PackageReference Include="LiteDB" Version="4.1.4" />
+        <PackageReference Include="System.IO.Hashing" Version="7.0.0" />
```

Comment:
Is the publisher Microsoft?

Reply:
Yes.
https://www.nuget.org/packages/System.IO.Hashing/7.0.0